pkg/tenant/members.go

```diff
@@ -384,8 +384,8 @@ func (s *Server) MemberRoleUpdate(c *gin.Context) {
 		}
 	}
 
-	// TOOD: Should we allow invitations to be updated?
-	if member.Status == db.MemberStatusPending {
+	// TOOD: Should we allow pending members to be role updated?
```

Comment:
Still kind of an open question - I'm not sure if it makes sense to allow the role to be updated for members who are still onboarding.